duckduckgo/Android
# Sync: Critical Path Tests

Task/Issue URL: https://app.asana.com/0/1174433894299346/1205433252919607

### Description
This PR adds the e2e test to verify that data can be synced across accounts.

### Steps to test this PR
Run maestro test locally, replacing `${CODE}` in `action_add_new_device` with a valid recovery code from the test account

https://github.com/duckduckgo/Android/actions/runs/7646940140/job/20836834057

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206229586525308
